src/config/file/v1.rs

```diff
@@ -0,0 +1,449 @@
+//! Version 1 of the configuration file.
```

Comment:
Is this version included in the config file somewhere? I didn't see it but it would be good if it was so that you can detect the version of the config file.